pages/evaluation-form/index.js

```diff
@@ -57,7 +57,8 @@ export default function EvaluationForm({ moods, setMoods }) {
 
   return (
     <>
-      <h2>Wie ist deine Stimmung heute?</h2>
+      <h2>Wie war die Session?</h2>
+      <h3>Wie ist deine Stimmung heute?</h3>
       <EmojiButton emoji="😤" onChange={handleMoodChange} />
       <EmojiButton emoji="😭" onChange={handleMoodChange} />
       <EmojiButton emoji="😩" onChange={handleMoodChange} />
```

Comment:
Sorry for this comment which is not on the new lines:

Since you are displayed a button with only an emoji, you need to add an `aria-label` to describe what's happening when the user clicks the button. #a11y

You could adapt your `EmojiButton` component to dynamically render the value of the `aria-label` based on the value of the `emoji` prop.

Reply:
No problem. Thank you very much for the advice. I will change this